components/integration/code/integration-bbe/resilience-engineering-built-in.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Resilience engineering, built-in'
-description: Ballerina’s built-in language and standard library features make programming in the failure-is-normal world of distributed systems as easy for every developer.
+description: Ballerina’s built-in language and library features make programming in the failure-is-normal world of distributed systems as easy for every developer.
```

Comment:
ditto

Reply:
Changed to: `language and library features of Ballerina`